server/issue.go

```diff
@@ -856,6 +808,70 @@ func getUpdateTask(database *api.Database, migrationType db.MigrationType, vcsPu
 	}, nil
 }
 
+// createDatabaseCreateTaskList returns the task list for create database.
+func (s *Server) createDatabaseCreateTaskList(ctx context.Context, c api.CreateDatabaseContext, instance api.Instance, project api.Project) ([]api.TaskCreate, []api.TaskIndexDAG, error) {
```

Comment:
Let's remove the return value []api.TaskIndexDAG because it's actually always nil.